addons/binding/org.openhab.binding.airquality/src/main/java/org/openhab/binding/airquality/internal/json/AirQualityJsonTime.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.airquality.internal.json;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link AirQualityJsonTime} is responsible for storing
+ * the "time" node from the waqi.org JSON response
+ *
+ * @author Kuba Wolanin - Initial contribution
+ */
+public class AirQualityJsonTime {
+
+    @SerializedName("s")
+    private String dateString;
+
+    @SerializedName("timeZone")
+    private String tz;
+
+    private final static SimpleDateFormat SDF = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+
+    /**
+     * Get Time zone from the JSON Response
+     * in following format: "+0100"
+     *
+     * @return {String}
+     */
+    public String getTimeZone() {
+        return tz.replace(":", "");
+    }
+
+    /**
+     * Get observation time
+     *
+     * @return {Calendar}
+     * @throws Exception
+     */
+    public Calendar getDateString() throws Exception {
+        Calendar calendar = Calendar.getInstance();
+        Date date = SDF.parse(dateString);
```

Comment:
Perhaps replace with:
```
Date date = SDF.parse(dateString + tz);
```
If adding the timezone to the formatter and the build string, shouldn't you get the observation time in the correct time zone?